utils/build/docker/nodejs/express4/app.js

```diff
@@ -44,7 +44,8 @@ app.get('/healthcheck', (req, res) => {
     library: {
       language: 'nodejs',
       version: require('dd-trace/package.json').version,
-      libddwaf_version: require('@datadog/native-appsec/package.json').libddwaf_version
+      libddwaf_version: require('dd-trace/node_modules/@datadog/native-appsec/package.json').libddwaf_version,
+      appsec_event_rules_version: require('dd-trace/packages/dd-trace/src/appsec/waf').wafManager?.rulesVersion
```

Comment:
btw i just realized, both infos are on the same object already, you can require it once and just use both props
```suggestion
      libddwaf_version: require('dd-trace/packages/dd-trace/src/appsec/waf').wafManager?.ddwafVersion,
      appsec_event_rules_version: require('dd-trace/packages/dd-trace/src/appsec/waf').wafManager?.rulesVersion
```

Reply:
With this change the `.npmrc` files are not longer needed too which is a nice plus.

Reply:
Niiice